### PR TITLE
Discovering the Lispworks IDE

### PR DESCRIPTION
**Pull request by vindarel**
Hi there, I explored the LispWorks IDE a bit. You who use it daily, what do you like most about it?

### CODE REVIEW

File: data-structures.md [hunk 1]
```diff
@@ -822,7 +822,45 @@ Hash Tables are created using the function
...
 `:test`, specifying the function used to test the equality of keys.
 
-_Note_: see shorter notations in the [Serapeum](https://github.com/ruricolist/serapeum/) or [Rutils](https://github.com/vseloved/rutils) libraries. For example, Serapeum has `dict`, and Rutils a `#h` reader macro.
+<div class="info-box info">
+<strong>Note:</strong> see shorter notations in the <a href="https://github.com/ruricolist/serapeum/">Serapeum</a> or <a href="https://github.com/vseloved/rutils">Rutils</a> libraries. For example, Serapeum has <code>dict</code>, and Rutils a <code>#h</code> reader macro.
+</div>
+
+<a name="add"></a>
+
+### Adding an Element to a Hash Table
```
Comment (phoe) [inline data-structures.md:831]: Generic hash-table stuff shouldn't really be a part of a Lispworks-related merge request, but that's just a nitpick from my side.

File: error_handling.md [hunk 2]
```diff
@@ -184,8 +184,12 @@ able to populate it with information to be consumed later:
 ;; #<MY-DIVISION-BY-ZERO {1005C18653}>
 ~~~
 
-Note: here's a quick reminder on classes, if you are not fully operational
-on the [Common Lisp Object System](clos.html):
+<div class="info-box info">
+<p>
+<strong>Note:</strong> here's a quick reminder on classes, if you are not fully operational
```
Comment (phoe) [inline error_handling.md:189]: Ditto.

File: files.md [hunk 3]
```diff
@@ -30,6 +32,9 @@ either `nil` if the file doesn't exists, or its
 [truename](http://www.lispworks.com/documentation/HyperSpec/Body/20_ac.htm)
 (which might be different from the argument you supplied).
 
+For more portability, use `uiop:probe-file*` or `uiop:file-exists-p`
```
Comment (phoe) [inline files.md:35]: Ditto.

File: files.md [hunk 4]
```diff
@@ -2,7 +2,9 @@
 title: Files and Directories
 ---
 
-Note: In this chapter, we use mainly
+We'll see here a handfull of functions and libraries to operate on files and directories.
```
Comment (phoe) [inline files.md:5]: s/handfull/handful/

**Comment by vindarel** [comment]
See it here: https://vindarel.github.io/cl-cookbook/lispworks.html

**Comment by svetlyak40wt** [comment]
Probably we should mention that Interface Builder is not available for OSX and on mobile platforms.

**Comment by svetlyak40wt** [comment]
Also, it would be useful to link back to https://github.com/fourier/awesome-lispworks repository.

**Comment by vindarel** [comment]
Thanks, will mention IB missing on OSX. Yep, I link back to awesome-lispworks already :)

**Comment by rprimus** [comment]
Wed Jun  2 07:22:58 BST 2021

Hi,

If you think it appropriate for this guide, the following link could be referenced for those that would like to change the Lispworks editor theme:

https://github.com/acelent/lw-editor-color-theme/blob/master/editor-color-theme.lisp

**Pull request merged**